pygmt/src/nearneighbor.py

```diff
@@ -84,9 +84,9 @@ def nearneighbor(data=None, x=None, y=None, z=None, **kwargs):
     x/y/z : 1d arrays
         Arrays of x and y coordinates and values z of the data points.
 
-    {I}
+    {spacing}
 
-    {R}
+    {region}
 
     search_radius : str
         Sets the search radius that determines which data points are considered
```

Comment:
> In a few functions and methods, blank lines are used between the different parameters. Currently, it appears unclear to me when and why this is done.

These blank lines are not necessary. They're there because these docstrings were written by different authors at different time.